CMakeLists.txt

```diff
@@ -11,6 +11,9 @@ project(networking-recipe VERSION 0.1 LANGUAGES C CXX)
 include(FindPkgConfig)
 include(CMakePrintHelpers)
 
+# Default: Release with Debug Info
+set(CMAKE_BUILD_TYPE "RelWithDebInfo")
```

Comment:
RelWithDbgInfo?

Reply:
Release with Debug Information. It sets `-O2 -g -DNDEBUG`.
Release sets `-O3 -DNDEBUG`.
Debug sets `-g`.